### PR TITLE
chore: add tests for tabs

### PR DESCRIPTION
**Pull request by imprashast**
Add tests for all components within tabs

### CODE REVIEW

File: test/wTabs.test.js [hunk 1]
```diff
@@ -40,3 +41,159 @@ describe('tabs', () => {
...
+    expect(screen.getByTestId('wunderbar')).toBeInTheDocument();
+  });
+
+  it('updates the wunderbar position on resize', () => {
+    const wunderbar = screen.getByTestId('wunderbar');
+    expect(wunderbar.style.left).toBe('0px');
+    expect(wunderbar.style.width).toBe('0px');
+    global.innerWidth = 800;
+    global.dispatchEvent(new Event('resize'));
+    waitFor(() => {
```
Comment (felicia-haggqvist) [inline test/wTabs.test.js:81]: Do we need to make this test asynchronous and await `waitFor`? It looks like they're awaiting `waitFor` in the [testing-library docs](https://testing-library.com/docs/dom-testing-library/api-async/#waitfor).

**Review by felicia-haggqvist — APPROVED** [review]
Just left one comment! Otherwise I think it looks good! 🎉

File: test/wTabs.test.js (hunk 1 above)
Comment (imprashast) [reply test/wTabs.test.js:81]: It somehow works tho 🤔

Comment (felicia-haggqvist) [reply test/wTabs.test.js:81]: Yeah that's why I wasn't sure if we need the await 😄

Comment (imprashast) [reply test/wTabs.test.js:81]: I'll leave it as it since it works 👍

**Pull request merged**